assets/js/main.js

```diff
@@ -49,60 +42,46 @@ var getClickEvent = function () {
   return 'ontouchstart' in document.documentElement ? 'touchend' : 'click';
 };
 
-/**
- * Opens and closes the mobile menu
- */
-var toggleMenu = function () {
-  $menuToggle.toggleClass('open');
-  $navigation.slideToggle(250);
-};
-
-$menuToggle.on(getClickEvent(), toggleMenu);
-
-// When a mobile nav link is clicked we should close the menu
-$('.header-nav-list-item').on(getClickEvent(), function () {
-  // We should be checking some state here instead of checking the width
-  if ($window.width() <= 770) {
-    toggleMenu();
-  }
-});
 
 $window.resize(function () {
-  // quick copy/paste, can be revisited for betterness
-
-  // brings back the nav if someone resizes the page
-  // needs work, still breaks with several click/resize combos
-  // probably unlikely that users will break it but should be fixed.
-  if ($(this).width() > 770 && $navigation.is(':hidden')) {
-    $navigation.show();
-  }
-
   // refetch transition heights on resize as well
   getTransitionHeights();
 }).scroll(function () {
   var s = window.scrollY;
 
-  if (s >= splashTransitionHeight - buffer && s < secondTransitionHeight) {
+  if (s >= splashTransitionHeight && s < secondTransitionHeight) {
     $colorChangeElements.addClass('after-fold');
-    setActiveNav($whoWeAreLink);
   }
   else if (s >= secondTransitionHeight && s < thirdTransitionHeight) {
     $colorChangeElements.addClass('after-fold');
-    setActiveNav($whatWeDoLink);
   }
   else if (s >= thirdTransitionHeight) {
     $colorChangeElements.addClass('after-fold');
-    $navLinks.removeClass('active');
   }
   else {
     $colorChangeElements.removeClass('after-fold');
-    $navLinks.removeClass('active');
   }
 });
 
+
 getTransitionHeights();
 
-$logo.click(function () {animateScroll(0)});
-$whoWeAreLink.click(function () {animateScroll(splashTransitionHeight)});
-$whatWeDoLink.click(function () {animateScroll(secondTransitionHeight + buffer)});
-$hireUsTriggers.click(function () {animateScroll(thirdTransitionHeight + buffer)});
+/* Contact Modal*/
+$(document).ready(function (){
```

Comment:
You dont need document ready unless you put the JS tag up in the `<head>` which you shouldnt do anyway. As long as the JS is at the end of the `<body>` (so right above `</body>`) the document will be ready as the browser rendered it all. So, here, remove the document ready lines since the script tag is in body